ee/be/modules/600-flant-integration/images/flant-pricing/hooks/node_group_resources_metrics.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+# Copyright 2024 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+for f in $(find /frameworks/shell/ -type f -iname "*.sh"); do
+  source $f
+done
+
+function __config__() {
+  cat << EOF
+    configVersion: v1
+    kubernetes:
+    - name: ngs
+      group: ngs
+      keepFullObjectsInMemory: false
+      queue: /node_group_resources_metrics
+      apiVersion: deckhouse.io/v1
+      kind: NodeGroup
+      jqFilter: |
+        {
+          "name": .metadata.name,
+          "labels": .spec.labels,
+          "taints": .spec.taints
+        }
+EOF
+}
+
+function __main__() {
+  ngs_capacity="$FP_NODE_GROUPS_CAPACITY"
+  group="group_node_group_resources_metrics"
+
+  jq -c --arg group "$group" '.group = $group' <<< '{"action":"expire"}' >> $METRICS_PATH
+
+  if ! context::has snapshots.ngs.0; then
+    return 0
+  fi
+
+  if [ "$ngs_capacity" == "" ]; then
+    return 0
+  fi
+
+  for node_group_name in $(context::jq -cr '.snapshots.ngs[].filterResult.name'); do
+    ng_capacity="$(jq -cr --arg ng_name "$node_group_name" '.[$ng_name] // ""' <<< "$ngs_capacity")"
+    if [ "$ng_capacity" == "" ]; then
+      continue
+    fi
+    cpu="$(jq -cr '.CPU' <<< "$ng_capacity")"
+    memory="$(jq -cr '.memory' <<< "$ng_capacity")"
+    labels="$(context::jq -cr --arg ng_name "$node_group_name" '.snapshots.ngs[].filterResult | select(.name == $ng_name) | ((.taints // [] | .[] | {("taint-"+.effect): .key}) // {}) as $t | (.labels // {}) as $l | $t * $l * {"name": .name}')"
```

Comment:
Do we need to export all labels? For what reason?

Reply:
Currently we only export the following labels:

```python
        labels = {
            "is_master": is_master,
            "is_system": is_system,
            "is_monitoring": is_monitoring,
            "is_frontend": is_frontend,
        }
```